README.md

```diff
@@ -10,6 +10,23 @@ You can install the command globally using:
 npm i -g @projectsophon/df-plugin-dev-server
 ```
 
+## Docker image installation and running
+
+You need to create network: `docker network create webproxy`
```

Comment:
I thought docker-compose can create networks?? cc @jacobrosenthal

Reply:
Yup, it can, I will change code to make it automatically